apps/agent/src/mastra/workflows/github-notify-reviewers.ts

```diff
@@ -0,0 +1,136 @@
+import { Step, Workflow } from '@mastra/core/workflows'
+import { z } from 'zod'
+import { discordClient } from '../../lib/discord'
+import { GITHUB_REPO, githubClient } from '../../lib/github'
+import { PullRequest } from '../../lib/type'
+import { formatDate } from '../../utils/datetime'
+import { DISCORD_GITHUB_MAP } from '../../constants/discord'
+
+class NotifyReviewersWorkflow {
+  private workflow: Workflow
+
+  constructor() {
+    this.workflow = new Workflow({
+      name: 'Notify Reviewers',
+    })
+  }
+
+  private getPendingReviews = new Step({
+    id: 'get-pending-reviews',
+    execute: async () => {
+      const prs = await githubClient.getRepoPRs(GITHUB_REPO, {
+        from: formatDate(new Date()),
+        isMerged: false,
+        isOpen: true,
+      })
+
+      // get PRs with reviewer assigned but not reviewed yet
+      const todayPRsWithReviewerAssigned = prs.filter((pr) => {
+        return pr.requested_reviewers && pr.review_comments === 0
```

Comment:
We should set a time limit for sending notifications. For instance, the system should send a notification only if there is no sign that the PR has been reviewed within 1 hour.